.github/workflows/build-test.yml

```diff
@@ -0,0 +1,41 @@
+name: build-test
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    strategy:
+      matrix:
+        node-version: [10.x]
```

Comment:
can we specify something like `node, lts/*` to test on the current stable, and latest LTS?

Reply:
and should we #443